src/utils/effigyAddressFormatter.ts

```diff
@@ -0,0 +1,16 @@
+import { ZERO_ADDRESS } from 'src/const/zeroAddress';
+
+export const effigyAddressFormatter = (address?: string) => {
+  const addressChunks = address?.split('...');
+  if (addressChunks?.length === 2) {
+    const addressPreChunksLength = addressChunks[0].length;
```

Comment:
We do have a formatter already named walletDigest

Reply:
This "effigyAddressFormatter" serves the purpose of creating an eth-address with the short versions of wallet-addresses. Also, it should adapt and create a valid eth wallet address and not be strictly tied to one format of input.
It should work with "0x123...456678" but also with a different example like "0x12345...678901".

Because, we have not adjusted the wallet-addresses on the backend, yet and without this extra function I am afraid it could break, once the backend response changes to the right format.

Reply:
yes, removed.

Reply:
@tcheee what do you mean?

Reply:
I was not clear sorry. We need it but I rename it so it is not confusing.